test/libsolidity/semanticTests/viaYul/function_selector.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
+    function f() external returns (bytes4) {
+        return this.f.selector;
+    }
+}
+// ====
+// compileViaYul: true
```

Comment:
Once the test works, you should change this to
```suggestion
// compileViaYul: also
```
Then it will validate against old code generation (but be careful while writing the tests: since isoltest will first only report on the run without yul, then run again and report on the run with yul, it can easily get confusing)